Output changes made.

ssailor96 opened the pull request:
Number of output files reduced from 2 to 1. Duplicate file names and number of duplicates are now printed to the console.

ssailor96 committed:
Output changes made.

Number of output files reduced from 2 to 1. Duplicate file names and number of duplicates are now printed to the console.

ssailor commented on duplicate_checker.py:
```diff
@@ -103,47 +103,36 @@ def dupeFinder(fileList):
+    else:
+        print("********************* Duplicate files found! Duplicates listed below *********************")
+        # print all duplicates to console along with the number of duplicates
+        numDuplicates = 0
```
I would use the len() method to print the number of duplicates. https://www.w3schools.com/python/ref_func_len.asp

ssailor96 replied:
Cannot use length function in this case, I am finding duplicates using the isDuplicate attribute.

ssailor replied:
That makes sense, sorry I missed that.

ssailor commented on duplicate_checker.py:
```diff
@@ -103,47 +103,36 @@ def dupeFinder(fileList):
-    mdFileName = "metadata_" + currentTime + ".json"
-    # mdFileName = "metadata_" + str(datetime.now()) + ".json"
+    # create file name with current datetime
+    outputFileName = "metadata_" + currentTime + ".json"
```
I recommend giving the output files a more meaningful name such as hash_report-xxxxx. I also recommend reformating the date that is generated to remove the periods. You can use the strftime method on a DateTime to print a format. I recommend using the format strftime(%Y-%m-%dT%H-%M-%S"). Using this I would expect the output name to be something like hash_report-2022-09-28T14-14-22.json.

ssailor96 replied:
The output file name format has been updated.

ssailor replied:
I ran this and can confirm, the output JSON filename is now what I expected.

ssailor commented:
You should add an entry to your .gitignore to ensure that output files you generate locally don't get checked into the repo. If you don't expect to have any json files that should be checked into the repo you can `*.json` to your .gitignore file and they will be automatically excluded from checkin.

ssailor96 committed:
Output file name updated

Made suggested changes to .gitignore file. Output file name format has been updated.

ssailor reviewed (approved):
Looks good now. Go ahead and merge this PR into dev
